SI/Sem.02/Pract.02/Task03.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+
+int main() {
+	double number1 = 0, number2 = 0;
+
+	std::cout << "Enter first number value: ";
+	std::cin >> number1;
+	std::cout << "Enter second number value: ";
+	std::cin >> number2;
+
+	number1 > number2 ? std::cout << "First number is larger than second number.\n" : std::cout << "Second number is larger than first number.\n";
```

Comment:
You can avoid repeating std::cout<< like this
std::cout<<(condition? "...." : ".....")

Reply:
Thanks, that's clever. I like it. I'm also disappointed that I didn't think of that on my own XD